homeassistant/components/media_player/kodi.py

```diff
@@ -110,7 +107,7 @@ def _get_players(self):
             return self._server.Player.GetActivePlayers()
         except jsonrpc_requests.jsonrpc.TransportError:
             if self._players is not None:
-                _LOGGER.warning('Unable to fetch kodi data')
+                _LOGGER.info('Unable to fetch kodi data')
```

Comment:
This is just a personal annoyance of mine. Since it's expected Kodi won't always be active, I don't think it needs to be a warning level log message. This can be reverted if others disagree.